torchbenchmark/models/detectron2_maskrcnn/__init__.py

```diff
@@ -0,0 +1,84 @@
+import torch
+import os
+import itertools
+import random
+import numpy as np
+
+# TorchBench imports
+from torchbenchmark.util.model import BenchmarkModel
+from torchbenchmark.tasks import COMPUTER_VISION
+
+# setup environment variable
+CURRENT_DIR = os.path.dirname(os.path.realpath(__file__))
+DATA_DIR = os.path.join(CURRENT_DIR, ".data", "coco2017-minimal")
+if not 'DETECTRON2_DATASETS' in os.environ:
+    os.environ['DETECTRON2_DATASETS'] = DATA_DIR
+
+from detectron2.config import instantiate
+from detectron2 import model_zoo
+from detectron2.utils.collect_env import collect_env_info
+from detectron2.utils.logger import setup_logger
+from detectron2.utils.events import EventStorage
+
+torch.manual_seed(1337)
+random.seed(1337)
+np.random.seed(1337)
+torch.backends.cudnn.deterministic = True
+torch.backends.cudnn.benchmark = False
+
+class Model(BenchmarkModel):
+    task = COMPUTER_VISION.DETECTION
+
+    def __init__(self, device=None, jit=False, train_bs=1, eval_bs=2):
```

Comment:
Does `bs` mean batch size? But in the code below they are not batch size, but the number of iterations to run train/eval.

Batch size for training is controlled in `data_cfg` like in Line46. We fixed eval batch size to 1.

Reply:
Thanks for pointing out, we will set `train.total_batch_size` to `train_bs`. Is there a way to setup eval batch size as well?

Reply:
> Is there a way to setup eval batch size as well?

We often don't do this but can enable it in D32330631

Reply:
If it can be enabled that will be very helpful, thanks!